synapse/storage/schema/__init__.py

```diff
@@ -109,7 +112,10 @@
     #
     # The threads_id column must written to with non-null values for the
     # event_push_actions, event_push_actions_staging, and event_push_summary tables.
-    74
+    #
+    # insertions to the column `full_user_id` of tables profiles and user_filters can no
+    # longer be null
+    75
```

Comment:
if I'm reading the mini changelog comment above properly, `SCHEMA_VERSION = 76` introduced these columns so should this not be bumped to 76?